pkg/parser/threefour/identity_backoff.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2020 Authors of Hubble
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package threefour
+
+import (
+	"sync"
+	"time"
+
+	"github.com/cilium/cilium/pkg/backoff"
+	hubtime "github.com/cilium/hubble/pkg/time"
+)
+
+const (
+	defaultBurst      = 25
+	defaultMinBackoff = 100 * time.Millisecond
+	defaultMaxBackoff = 5 * time.Second
```

Comment:
not super confident about these default settings, open to suggestions for burst/min/max

Reply:
max backoff can be larger i think, maybe 1 minute?

the most common scenario for missing identity i believe is when there are "stale" flows that contain deleted identities. in this case the identity will never be resolved, and we are basically waiting for stale flows to go away. i'm even fine without the default burst and immediately starting to back off :)